tests/test_downscoped.py

```diff
@@ -496,6 +504,20 @@ def test_default_state(self):
         assert not credentials.expired
         # No quota project ID set.
         assert not credentials.quota_project_id
+        assert credentials.universe_domain == DEFAULT_UNIVERSE_DOMAIN
+
+    def test_create_with_customized_universe_domain(self):
```

Comment:
Can you add more tests? e.g. that request is sent to STS with the custom universe domain